Task1/scripts/Utils/DataBinding.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Class representing binding example
+ */
+export default class DataBinding {
```

Comment:
Can you please explain logic behind this class? Why 'ing if it is a class, better name would be a noun, e.g. DataBinder.